network-api/networkapi/wagtailpages/pagemodels/products.py

```diff
@@ -118,8 +120,12 @@ class ProductPage(FoundationMetadataPageMixin, Page):
     ProductPage is the superclass that SoftwareProductPage and
     GeneralProductPage inherit from. This should not be an abstract
     model as we need it to connect the two page types together.
+
+    ProductPage is re-using the product_page.html template that PNI Products use.
```

Comment:
Let's do this a little differently: in order to keep the old app working, we want to put and templates that we need to change in their own dir, and point to those, so that even if we mess this work up (let's hope we don't of course!) the old app won't break when we push the changes, and we don't have to lock prod pushing for as long as we need to fix whatever we got wrong.